front/app/containers/IdeasShow/index.tsx

```diff
@@ -140,20 +140,17 @@ export const IdeasShow = ({
               translateButtonClicked={translateButtonIsClicked}
             />
           </Box>
-          {compact &&
-            participationContext?.attributes.participation_method !==
-              'voting' && // To reduce bias we want to hide the author data during voting methods
-            statusId && (
```

Comment:
We actually already had this check within the MetaInformation component as well, which hides the author. So this check here is unnecessary and also results in the rest of the metadata being incorrectly hidden.